tests/integration/lsp_tests.rs

```diff
@@ -14240,7 +14247,7 @@ fn lsp_deno_json_workspace_node_modules_dir() {
     "project1/deno.json",
     json!({
       "workspace": ["project2"],
-      "nodeModulesDir": true,
+      "nodeModules": "local-auto",
```

Comment:
I think we should change this to `"nodeModules": "auto"`. This is too much to type.